index.bs

```diff
@@ -5743,32 +5743,28 @@ To <dfn>deserialize header</dfn> given |protocol header|:
 
 <pre class="cddl local-cddl">
 network.Initiator = {
-    type: "parser" / "script" / "preflight" / "other",
     ? columnNumber: js-uint,
```

Comment:
it's a bit awkward that everything is optional but the initiator field itself on network.BeforeRequestSentParameters is not.

Reply:
Yeah, maybe it should be now if there's no stack.

Reply:
Updated with that change.